server/opendp_apps/analysis/tools/stat_spec.py

```diff
@@ -82,7 +83,11 @@ def __init__(self, props: dict):
 
         self.categories = self.variable_info.get('categories')
         self.var_type = self.variable_info.get('type')
-
+        # print("VAR TYPE", self.var_type)
+        if self.var_type == VAR_TYPE_INTEGER:
```

Comment:
This should go into the dp_histogram_spec, e.g. under `run_01_initial_handling` or another abstract method.
(fyi: I'll make this change while testing)